app/src/main/res/layout/activity_camera.xml

```diff
@@ -0,0 +1,143 @@
+        android:layout_width="@dimen/zero"
+        android:layout_height="@dimen/zero"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <co.condorlabs.customcomponents.imagecropview.AppCompatCropImageView
+        android:id="@+id/photoCaptured"
+        android:layout_width="@dimen/zero"
+        android:layout_height="@dimen/zero"
+        android:adjustViewBounds="true"
+        android:background="@android:color/transparent"
+        android:scaleType="centerCrop"
+        app:cornerColor="@android:color/white"
+        app:cornerSize="@dimen/camera_corner_size"
+        app:edgeColor="@android:color/white"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:linkedView="@+id/vRectangle"
+        app:outsideCropColor="@color/image_crop_view_outside_background_color"
+        app:resizeCornerDrawable="@drawable/ic_ball" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="@dimen/zero"
+        android:layout_height="wrap_content"
+        android:fitsSystemWindows="true"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="@dimen/zero"
+            android:layout_height="wrap_content"
+            android:background="@color/background_black_with_with_40_percent_transparency"
+            android:padding="@dimen/camera_footer_default_padding"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <TextView
+                android:id="@+id/capturePhotoDescription"
+                style="@style/camera_take_photo_description_style"
+                app:layout_constraintBottom_toBottomOf="@id/descriptionAuxBarrier"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <androidx.constraintlayout.widget.Barrier
+                android:id="@+id/descriptionAuxBarrier"
+                android:layout_width="@dimen/zero"
+                android:layout_height="wrap_content"
+                app:barrierDirection="bottom"
+                app:constraint_referenced_ids="capturePhotoDescription" />
+
+            <co.condorlabs.customcomponents.custombutton.CustomButton
+                android:id="@+id/cancelPhoto"
```

Comment:
This views should start with bt -> btCancel

Reply:
Ok